src/components/styles-provider/README.md

```diff
@@ -0,0 +1,20 @@
+# StylesProvider
+
+_This is an **experimental** component and might change in the future. Use with caution!_
+
+## Description
+
+Applies some global styles according to our design system conventions. This component should be rendered as one of the first components in our application tree.
+
+## Usage
+
+```js
+import { StylesProvider } from '@commercetools-frontend/ui-kit';
+```
+
+## Properties
+
+| Props          | Type               | Required | Values | Default | Description                                                                                      |
+| -------------- | ------------------ | :------: | ------ | ------- | ------------------------------------------------------------------------------------------------ |
+| `baseFontSize` | `PropTypes.string` |    -     | -      | `1rem`  | Defines the font size baseline from which the `rem` font sizes of the application are based from |
```

Comment:
It doesn't make sense to set baseFontSize to `1rem`. It should be a pixel value. Maybe name it something like `bodyFontSize` or `htmlFontSize` to imply that it's being set on the body...

Reply:
Sure, I wanted to discuss this with you tomorrow.